server/mockAPI.js

```diff
@@ -1069,6 +1069,42 @@ const mockMeasurements = [
   {
     'size': -1,
     'running': false,
+    'country_code': 'IT',
+    'test_name': 'dash',
+    'test_start_time': '20170114T080500Z',
+    'completed': true,
+    'keep': false,
+    'asn': 'AS123',
+    'stale': false,
+    'anomaly': false,
+    'id': '20170114T080500Z-IT-AS123-dash',
+    'result': 'ok',
+    'deck_id': 'web',
+    'summary': {
+      'test_start_time': '2017-01-12 00:29:22',
+      'anomaly': false,
+      'country_code': 'GR',
+      'asn': 'AS10',
+      'test_name': 'ndt',
```

Comment:
The three lines above should probably be consistent with what is written before summary (i.e. `country_code: IT`, `asn: AS123`, `test_name: dash`).